packages/commons/src/fhir/location.ts

```diff
@@ -121,6 +121,10 @@ export function isHealthFacility(
   return location.type?.coding?.[0].code === 'HEALTH_FACILITY'
 }
 
+export function isOffice(location: Location): location is HealthFacility {
```

Comment:
```suggestion
export function isOffice(location: Location): location is CRVSOffice {
```
or something similar?

Reply:
Whoops. Thanks for that catch